documentation/how-to/upgrade-server.md

```diff
@@ -44,3 +44,14 @@ _Warning: It is essential that upgrades are done one consecutive revision at a
 time. Charmed Temporal K8s can only guarantee backward compatibility between two
 consecutive revisions in line with the upgrade system adopted by the Temporal
 Server._
+
+## Appendix
+
+The table below shows a mapping between the Temporal K8s charms and the Temporal
+server versions. It can be used as a reference for upgrading your charm
+revisions in line with the Temporal server version so as to avoid any breaking
+changes.
+
+| Temporal Server Charm Revision | Temporal Admin Charm Revision | Temporal Server Version |
+| :----------------------------: | :---------------------------: | :---------------------: |
+|             20-21              |              8-9              |         v1.21.5         |
```

Comment:
Can you maintain this mapping? For example if you release a revision that needs to be ignored for some reason then you lose whatever mapping you had (not sure if you're implying anything specific with this mapping)

Reply:
One thing you can also do btw is add a version to your charm if you want to indicate which upstream Temporal version it related to. That can be done as follows https://discourse.charmhub.io/t/file-version/7155 - I think the bit about auto-detecting the version based on VCS is false according to [this thread](https://github.com/canonical/charmcraft/issues/37) but I think you can still populate a `version` file during the charm build that charmcraft will include in the charm.

Reply:
The mapping is useful for users who want to deploy/upgrade the charm to know what version of Temporal server they would be upgrading to with the given charm revision. I've also included a workload version that will be visible when you run `juju status`, that can be updated when the charm uses a new version of the Temporal server.